setup.py

```diff
@@ -1,42 +1,57 @@
 from setuptools import setup, find_packages
 from os import getcwd
+
 current_path = getcwd()
 
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
 requirements = [
     "sympy>=1.10.1",
-    "numpy==1.22.3",
-    "networkx==2.8",
+    "numpy>=1.22.3",
+    "networkx>=2.8",
     "scipy==1.8",
```

Comment:
Hm should we also just do ">=" for the remaining packages or leave it as it is unless the other packages also throw incompatibility errors?

Reply:
As far as I could test currently only  `scipy` requires the exact version. Have I forgotten to put the `>=` for any other?